ansible/roles/debops.slapd/files/etc/ldap/schema/debops/postfix.schema

```diff
@@ -0,0 +1,56 @@
+# postfix.schema - Postfix schema definition
+
```

Comment:
The schema you proposed is very basic, and I would like to include a much more expansive one, that supports mail routing via LDAP attributes, mail groups, etc. The closest I would see to a working schema definition is [50ns-mail.ldif](https://pagure.io/389-ds-base/blob/master/f/ldap/schema/50ns-mail.ldif) which is somewhat based on the [Netscape mailGroup schema](https://ldapwiki.com/wiki/Original%20Mailgroup%20Schema%20From%20Netscape), but of couse it would have to be rewritten in the `.schema` format for DebOps. The [draft-lachman-ldap-mail-routing-03.txt](https://tools.ietf.org/html/draft-lachman-ldap-mail-routing-03) RFC also has some interesting ideas, but I'm not sure how compatible they are with the Netscape schema. The [Enterprise mail routing with Postfix and LDAP](https://blog.felipe-alfaro.com/2006/05/24/enterprise-mail-routing-with-postfix-and-ldap/) might also be worth a read.

In the end I'm not ure if we can use the OIDs from `50ns-mail.ldif` schema and just extend it for our needs, or go with a separate OID namespace based on DebOps OID registration. Still, since messing with already imported LDAP schemas in OpenLDAP is hard bordering on impossible, this inclusion would probably be worth a separate pull request so that it can be worked on separately from the rest of Postfix-based code and thoughly engineered.

Looking at Dovecot and other core mail services during schema creation might also be worth it. Roundcube probably doesn't really need all that much unique objects or attributes, not sure about other services.